cap/jsonschemas/options/deposits/records/lhcb-v0.0.1.json

```diff
@@ -178,417 +178,230 @@
-          },
-          {
-            "key": "dst_selection.code.platform",
-            "placeholder": "E.g. x86_64-slc5-gcc46-opt"
+            "key": "ntuple_userdst_production[].davinci_script._type",
+            "title": "Please select the preferred way to upload a file",
+            "type": "radios",
+            "titleMap": [
+              {"value": "external_url", "name": "External URL"},
+              {"value": "file_upload", "name": "File Upload"}
+            ]
           },
           {
-            "key": "dst_selection.code.user_code",
+            "key": "ntuple_userdst_production[].davinci_script.source",
+            "condition": "model['ntuple_userdst_production'][arrayIndices[0]]['davinci_script']['_type'] == 'external_url'",
             "items": [
               {
-                "key": "dst_selection.code.user_code[].url",
-                "placeholder": "E.g. svn@svnweb.cern.ch/cern/wsvn/myrepo"
-              },
-              {
-                "key": "dst_selection.code.user_code[].description",
-                "placeholder": "E.g. Please enter the user code description, location, etc"
+                "key": "ntuple_userdst_production[].davinci_script.source.preserved",
+                "destroyStrategy": "retain",
+                "type": "radios",
+                "titleMap": [
+                  {"value": true, "name": "Yes"},
+                  {"value": false, "name": "No"}
+                ]
               },
               {
-                "key": "dst_selection.code.user_code[].instructions",
-                "type": "textarea",
-                "fieldHtmlClass": "custom-textarea",
-                "placeholder": "E.g. Please enter the instructions for using the code, e.g. Open 'README' file"
+                "key": "ntuple_userdst_production[].davinci_script.source.url",
+                "notitle": true,
+                "destroyStrategy": "retain",
+                "placeholder": "E.g. https://git@github.com:johndoe/myrepo.git"
               }
             ]
+          },
+          {
+            "key": "ntuple_userdst_production[].davinci_script.key",
+            "condition": "model['ntuple_userdst_production'][arrayIndices[0]]['davinci_script']['_type'] == 'file_upload'",
+            "type": "file_selector",
+            "title": "Select file"
           }
         ]
       },
       {
-        "key": "dst_selection.production_info",
+        "key": "ntuple_userdst_production[].ganga_script",
+        "title": "Ganga script",
         "type": "fieldset",
         "items": [
           {
-            "key": "dst_selection.production_info.data",
-            "startEmpty": true,
-            "items": [
-              {
-                "key": "dst_selection.production_info.data[].location",
-                "placeholder": "E.g. sim://LHCb/Collision12/Beam4000GeV-VeloClosed-MagDown/RealData/Reco14/Stripping20/90000000 ( Full stream )/BHADR",
-                "type": "cap:formAutofill",
```

Comment:
new field _stripping_turbo_selection[].bookkeping_location_ should have _type_ and _ref_ properties like here